JEasyCrypto/src/easycrypto/ROT13Method.java

```diff
@@ -0,0 +1,39 @@
+package easycrypto;
+
+import easycrypto.EasyCryptoAPI.Result;
+import easycrypto.EasyCryptoAPI.ResultCode;
+
+class ROT13Method implements CryptoMethod {
+
+    @Override
+    public Result encrypt(final String toEncrypt) {
+        String toStoreTo = toRot13(toEncrypt);
+        return new Result(ResultCode.ESuccess, toStoreTo);
+    }
+
+    @Override
+    public Result decrypt(final String toDecrypt) {
+        String toStoreTo = toRot13(toDecrypt);
+        return new Result(ResultCode.ESuccess, toStoreTo);
+    }
+
+    @Override
+    public String method() {
+        return "rot13";
+    }
+
+    private String toRot13(final String input) {
+        return input.codePoints().map(c -> {
+                    if ((c >= 'a' && c <= 'm') || (c >= 'A' && c <= 'M')) {
+                        return c + 13;
+                    } else if ((c >= 'n' && c <= 'z') || (c >= 'N' && c <= 'Z')) {
+                        return c - 13;
+                    } else {
+                        return c;
```

Comment:
This `else` statement can only be reached if we enter non-alphabetical characters. So maybe at this point there should be an error message, because if this `else` is reached, the character is not going to be encrypted.

Reply:
Now there's a warning message at that point as well.

Reply:
Thank you! Looks good!